syntax = "proto3";

import "inventory.proto";

service InventoryService {
    rpc CreateBook (Book) returns (CreateBookResponseStatus) {}
    rpc GetBook (ISBN) returns (GetBookResponseStatus) {}
}